crates/vm/levm/src/vm.rs

```diff
@@ -721,6 +731,7 @@ impl VM {
                     .push(U256::from(SUCCESS_FOR_CALL))?;
             }
             TxResult::Revert(_) => {
+                // self.env.refunded_gas = self.env.refunded_gas.checked_sub(tx_report.gas_refunded.into()).ok_or(VMError::Internal(InternalError::ArithmeticOperationUnderflow))?;
```

Comment:
I left it as a possible change after discuss it with Jere, since we are not reverting the refunded gas in case of the called callframe generates an error, but it can be deleted.

Reply:
Ok, I'd add a TODO comment there for context.

Reply:
Was done in [Add todo comment for revert refunded gas](https://github.com/lambdaclass/ethrex/pull/1281/commits/5e2c8be17a47673ed738571ca32547e73020b9b7) and reverted in [Remove TODO comment](https://github.com/lambdaclass/ethrex/pull/1281/commits/30e6f17063d07f30bc1bc816715983637c8006f3)